frontend/src/store/playbackManager.ts

```diff
@@ -540,7 +540,12 @@ class PlaybackManagerStore {
    * Updates mediasession metadata based on the currently playing item
    */
   private _updateMediaSessionMetadata = (): void => {
-    if (!isNil(this.currentItem)) {
+    if (isNil(this.currentItem)) {
```

Comment:
I think this whole block can be removed and just leave what is in the `else` part? This seems redundant to me.

Reply:
Indeed, refactored